packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -251,14 +277,10 @@ contract Deploy is Deployer {
 
         // Set up the Superchain if needed.
         if (_needsSuperchain) {
-            setupSuperchain();
+            deploySuperchain();
         }
 
-        if (cfg.useInterop()) {
-            deployImplementationsInterop();
-        } else {
-            deployImplementations();
-        }
+        deployImplementations({ _isInterop: cfg.useInterop() });
```

Comment:
Passing in this arg is somewhat overkill, as `cfg` is already available in the `deployImplementations` function, but my goal is to have all of the `cfg.use_()` flags visible here rather than buried in function calls.

Reply:
Within `deployImplementations` should we assert consistency, i.e. `require(_isInterop == cfg.useInterop())`?

Reply:
9862f32853e1